src/pages/lessons/projects/3.mdx

```diff
@@ -91,14 +91,14 @@ tiered NFTs and uncover the unique superpowers they possess. By the end of this
 tutorial, you will have gained a wealth of knowledge and accomplished the
 following steps:
 
-- Setting up the development environment
-- Writing and deploying a Solidity smart contract
-- Creating tiered NFTs with varying levels of rarity and attributes
-- Write scripts for deployment
-- Exploring additional functionalities and possibilities with tiered NFTs ... in
+- Set up a development environment
+- Write and deploy a Solidity smart contract
+- Create tiered NFTs with varying levels of rarity and attributes
+- Write a script for deployment
+- Explore additional functionalities and possibilities with tiered NFTs... in
```

Comment:
I guess it's personal taste with the verb form. 

In my opinion, from a motivational perspective the continuous form is a nice touch for beginners here in the lead-in,  who might encounter some or all of these tasks as adventures in themselves, especially if they run into trouble i.e. create a bit of a safe space. 

We had made an explicit effort to actually rename the sub-sections in the way you're suggesting here (the conversation back then was to get learners used to open source conventions such as succinctly naming Github commits, etc). Is that the reason for homogenising them i.e. with the sub-headers in the lesson? 

I can go with it, but thought I'd share my rationale.

Reply:
only motivation here was consistent use of language. in this case, the odd one out was "Write scripts...", which could be converted to "Writing scripts..."